### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by Wuon**
### Related Issue(s)

resolves #

### CODE REVIEW

File: client/src/pages/inventory/VariantCreate.tsx [hunk 1]
```diff
@@ -1,51 +1,209 @@
...
+                required: true,
+                minLength: 1,
+              }}
+              render={({
+                field: { onChange, value },
+                fieldState: { invalid, error },
+              }) => (
+                <TextField
+                  fullWidth
+                  label="Variant Descriptor (i.e 500ml, blue)"
```
Comment (callmekungfu) [inline client/src/pages/inventory/VariantCreate.tsx:105]: So _i.e._ in English means "in another word", what you are looking for is _e.g._. But I recommend that you just omit it, people will understand.

https://www.warnell.uga.edu/sites/default/files/inline-files/ieVeg.pdf

TL;DR:

> I find that sometimes students are unsure about when to use the abbreviations “i.e.” and 
“e.g.” and the difference between the two. They are not interchangeable; each has its own 
meaning and usage. The abbreviation “i.e.” stands for id est, which is Latin for “that is.” The 
abbreviation “e.g.” stands for the Latin phrase exempli gratia, meaning “for example.”

**Review by callmekungfu — APPROVED** [review]

**Review by callmekungfu — APPROVED** [review]
